libs/spark/src/Unstable_SectionMessage/Unstable_SectionMessage.tsx

```diff
@@ -0,0 +1,211 @@
+  | 'message'
+  | 'action';
+
+const useStyles = makeStyles<Unstable_SectionMessageClassKey>(
+  (theme) => ({
+    root: (props: Unstable_SectionMessageProps) => ({
+      borderRadius: 4,
+      display: 'flex',
+      gap: 16,
+      padding: 24,
+      borderWidth: 1,
+      borderStyle: 'solid',
+      ...(props.severity === 'error' && {
+        backgroundColor: theme.unstable_palette.red[100],
+        borderColor: theme.unstable_palette.red[700],
+      }),
+      ...(props.severity === 'info' && {
+        backgroundColor: theme.unstable_palette.blue[100],
+        borderColor: theme.unstable_palette.blue[600],
+      }),
+      ...(props.severity === 'success' && {
+        backgroundColor: theme.unstable_palette.green[100],
+        borderColor: theme.unstable_palette.green[600],
+      }),
+      ...(props.severity === 'warning' && {
+        backgroundColor: theme.unstable_palette.yellow[100],
+        borderColor: theme.unstable_palette.yellow[600],
+      }),
+    }),
+    icon: (props: Unstable_SectionMessageProps) => ({
+      fontSize: theme.unstable_typography.pxToRem(24),
+      lineHeight: 1,
+      color: theme.unstable_palette.text.icon,
+      ...(props.severity === 'error' && {
+        color: theme.unstable_palette.red[700],
+      }),
+      ...(props.severity === 'info' && {
+        color: theme.unstable_palette.blue[600],
+      }),
+      ...(props.severity === 'success' && {
+        color: theme.unstable_palette.green[600],
+      }),
+      ...(props.severity === 'warning' && {
+        color: theme.unstable_palette.yellow[600],
+      }),
+    }),
+    message: {
+      ...theme.unstable_typography.body,
+      color: theme.unstable_palette.text.body,
+      flexGrow: 2,
+    },
+    action: { alignSelf: 'flex-start', justifySelf: 'flex-end' },
+  }),
+  { name: 'MuiSparkUnstable_SectionMessage' }
+);
+
+const defaultIconMapping = {
+  success: <Unstable_CheckCircle2 />,
+  warning: <Unstable_AlertTriangle />,
+  error: <Unstable_AlertOctagon />,
+  info: <Unstable_Info />,
+};
+
+const Unstable_SectionMessage = React.forwardRef(
+  function Unstable_SectionMessage(props: Unstable_SectionMessageProps, ref) {
+    const {
+      action,
+      children,
+      classes: classesProp,
+      className,
+      icon,
+      onClose,
+      title,
+      role = 'alert',
+      severity = 'info',
+      ...other
+    } = props;
+
+    const classes = useStyles({ severity });
+
+    return (
+      <Paper
+        role={role}
+        square
+        elevation={0}
+        className={clsx(classes.root, className)}
+        ref={ref}
+        {...other}
+      >
+        {icon !== false ? (
+          <div className={clsx(classesProp?.icon, classes.icon)}>
+            {icon || defaultIconMapping[severity]}
+          </div>
+        ) : null}
+
+        <div className={clsx(classesProp?.message, classes.message)}>
+          {title ? (
+            <Unstable_SectionMessageTitle>{title}</Unstable_SectionMessageTitle>
```

Comment:
```suggestion
            <div className={clsx(classes.title, className)}>{title}</div>
```
(Part of combining into one component)